steps/scan-with-appsweep/1.0.0/step.yml

```diff
@@ -0,0 +1,60 @@
+title: AppSweep Mobile Security Testing
+summary: Scan your Android app using Guardsquare's mobile application security testing
+  tool AppSweep
```

Comment:
No need to add a line break here, it can be a single line.

Reply:
this is a bit weird, in our repo this is on one line, but when creating this in the steplip using `bitrise share` it adds this line break.

Reply:
Oh, then please ignore it, we'll investigate this.